blocks/columns/columns.css

```diff
@@ -15,6 +29,11 @@ main .section-top-grid > div > div > h1 {
     padding: 0;
     }
 
+.teaser-list-manual .teaser-list div div {
+    padding: var(--spacer-element-10) 0;
+    border-bottom: 1px solid var(--neutral-sand);
+    }
```

Comment:
Is this similar to existing `divider` functionality but within a column block? Did you try using `divider`?